package.json

```diff
@@ -1,16 +1,30 @@
 {
```

Comment:
not a big deal, but this doesnt seem to related to adding this new challenge solution?  Would be best to put this into its own PR but it dont really matter.